tests/integration/test_charm.py

```diff
@@ -19,6 +20,7 @@
 MEM_1G_APP_NAME = f"{APP_NAME}-1g"
 MEM_1G_BUILDPACK_APP_NAME = f"{BUILDPACK_APP_NAME}-1g"
 ALL_APP_NAMES = [APP_NAME, BUILDPACK_APP_NAME, MEM_1G_APP_NAME, MEM_1G_BUILDPACK_APP_NAME]
+NGINX_INGRESS_NAME = "nginx-ingress-integrator"
```

Comment:
Rename to `INGRESS_NAME` to make it generic, will be usefull when implementing traefik